tacs/pytacs.py

```diff
@@ -513,6 +513,43 @@ def getCompNames(self, compIDs=None):
 
         return compDescripts
 
+    def getNodesForCompIDs(self, compIDs=None):
```

Comment:
This is returning the local node ordering (how the nodes are split across procs) right? 

If so, can we change this function name to: `getLocalNodeIDsForComps`?

Reply:
I believe it actually returns the global node numbering. It was my understanding that TACS only used global numbering for nodes and doesn't renumber when partitioning occurs. However, maybe I am confusing that with a previous version of the code or something else entirely? 

If you can think of an easier way to directly get the local node ordering, then I can update the function. Local numbering would probably be easier to work with. The way I have been doing that on my own requires calling `getOwnerRange()` from the assembler and using the rank get the local numbering. But that conversion process I have been doing outside of `getNodesForCompIDs()`. If it makes more sense to do that in the body of the function, then I can add it there.